hotham/Cargo.toml

```diff
@@ -41,6 +41,7 @@ renderdoc = "0.10"
 
 [dev-dependencies]
 approx = "0.5"
+lenient_bool = "0.1.1"
```

Comment:
New dependency alert! Isn't is sufficient to just check to see if the environment variable is defined? Who cares what the value is.

Reply:
Users of hotham will not be impacted because it's only a dev decency. It felt more proper to respect the value of the environment variable but it doesn't do much, if any, difference to the current usage. I can remove it if you feel strongly about it.

Reply:
You don't even need to feel strongly about it. It was easy to remove and replace it with a test for the truthy strings.

Reply:
Thank you! Yeah because it's just a testing thing I felt like actually checking the value of the var was overkill, but this is a good solution 👌